python/ola/PidStore.py

```diff
@@ -160,6 +160,21 @@ def ValidateAddressing(self, args, command_class):
   def __cmp__(self, other):
     return cmp(self._value, other._value)
 
+  def __eq__(self, other):
```

Comment:
Related comment, this is only checking value, not other fields.

Reply:
I copied what `__cmp__` did. Maybe not the best choice.